README.md

```diff
@@ -12,7 +13,9 @@ Even though the architecture changed significantly with the 2.0 release of QUARK
 Documentation with a tutorial and developer guidelines can be found here: https://quark-framework.readthedocs.io.
 
 ## Prerequisites
-As this framework is implemented in Python 3.9, you need to install this version of Python if you do not already have it installed. Other versions could cause issues with other dependencies used in the framework. Additionally, we rely on several pip dependencies, which you can install in two ways:
+As this framework is implemented in Python 3.9, you need to install this version of Python if you do not already have it installed.
+Other versions could cause issues with other dependencies used in the framework.
+Additionally, we rely on several pip dependencies, which you can install in two ways:
 
 1. Install pip packages manually, or
 2. Use the QUARK installer.
```

Comment:
Could you also make sure that there is an empty line before the listing of packages. Otherwise they seem not to be displayed properly. I overlooked that in your last PR, I think.